lib/npm.js

```diff
@@ -18,23 +21,30 @@ const env = Object.keys(process.env).reduce((e, key) => {
 
 module.exports.install = install;
 function install (deps = [], opts = {}) {
-  if (!deps || !deps.length) {
+  if (deps !== null && (!deps || !deps.length)) {
```

Comment:
its a bit weird to special case `null` vs other falsy values

Reply:
Weird but intentional to minimize a technically breaking api change. Before this it would consider any falsy values an error. The new install is calling it without deps, and I considered changing it to an empty array but making that not-error also felt like a breaking change. I also considered passing a magic string like "i promise I know what I am doing" (joking, but any magic string felt like that).

Explicitly passing `null` felt like one of the least breaking options to me. Any better idea for keeping the api contract as intact as possiable?

Reply:
yeah, that's fair about the breaking change aspect, maybe the best solution given the options